README.md

```diff
@@ -16,6 +16,7 @@ The different parameters of the task are explained below:
 * **Authentication**: Select the authentication mode for connecting to the SQL Server. In Windows authentication mode, build service account, is used to connect to the SQL Server. In SQL Server Authentication mode, the SQL login and Password have to be provided in the parameters below.
 * **SQL User name**:  Provide the SQL login to connect to the SQL Server. The option is only available if SQL Server Authentication mode has been selected.
 * **SQL Password**: Provide the Password of the SQL login. The option is only available if SQL Server Authentication mode has been selected.
+* **Initial Catalog**: SSISDB catalog name for the Integration Services.  Default to **SSISDB**
```

Comment:
Should this be a master, as master was the default value I was passing in previously, in order to maintain compatibility?
https://github.com/mmajcica/DeploySsis/blob/ee5b285f998f7cd050e56f7c12bea520dead63b5/task/ps_modules/ispac.psm1#L9

Reply:
Got it.  It should be 'master' so it will not fail if SSISDB does not exists.